base_layer/wallet_ffi/python/tari_wallet/wallet_helper.py

```
@@ -0,0 +1,283 @@
+"""
+Wallet Helper Functions
+
+This module provides high-level helper functions for wallet creation with 
+automatic base node discovery integration.
+"""
+
+import tempfile
+import os
+from typing import Optional, Dict, Any, Callable, List
+from .network import TariNetwork, NetworkManager
```

Ruff (0.11.9)

`.network.TariNetwork` imported but unused

Remove unused import: `.network.TariNetwork`

(F401)